index.js

```diff
@@ -38,6 +38,20 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('pre-release', {
+    alias: 'p',
+    describe: 'Should the release be a pre-release?',
+    type: 'boolean',
+    default: false,
+    global: true
+  })
+  .option('tag', {
```

Comment:
Sorry, I think this should be called "Prerelease Identifier". I'm bad at naming so just my own opinion...